subsys/bluetooth/mesh/prov.c

```diff
@@ -239,11 +308,15 @@ int bt_mesh_input_string(const char *str)
 {
 	BT_DBG("%s", log_strdup(str));
 
+	if (strlen(str) > PROV_IO_OOB_SIZE_MAX) {
```

Comment:
Should this also check against `bt_mesh_prov->input_size`?

Reply:
Nop. This will not work since the function is used for both provisionee and provisioner. Provisioner will use it for output string OOB. More correctly to use here `bt_mesh_prov_link.oob_size`. However, I see oob fields are not set for provisionee site. Will fix.